code/modules/surgery/organs/eyes.dm

```diff
@@ -140,3 +139,181 @@
 
 /obj/item/organ/eyes/robotic/shield/emp_act(severity)
 	return
+
+#define RGB2EYECOLORSTRING(definitionvar) ("[copytext(definitionvar,2,3)][copytext(definitionvar,4,5)][copytext(definitionvar,6,7)]")
+
+/obj/item/organ/eyes/robotic/glow
+	name = "High Luminosity Eyes"
+	desc = "Special glowing eyes, used by snowflakes who want to be special."
+	origin_tech = "material=3;biotech=3;engineering=3;magnets=4"
+	var/current_color_string = "#ffffff"
+	eye_color = "000"
+	var/active = FALSE
+	actions_types = list(/datum/action/item_action/organ_action/use, /datum/action/item_action/organ_action/toggle)
+	var/olddir = 0
```

Comment:
0 is a valid dir of sorts, atleast, it has a useful function in ```get_step()``` (it's what we use for ```get_turf()```, ```#define get_turf(A) (get_step(A, 0))```)